src/sidebar/components/Annotation/AnnotationShareInfo.js

```diff
@@ -1,15 +1,14 @@
 import { Icon, Link } from '@hypothesis/frontend-shared';
-
-import { useStoreProxy } from '../../store/use-store';
-import { isPrivate } from '../../helpers/permissions';
+import classnames from 'classnames';
 
 /**
- * @typedef {import("../../../types/api").Annotation} Annotation
+ * @typedef {import("../../../types/api").Group} Group
  */
 
 /**
  * @typedef AnnotationShareInfoProps
- * @prop {Annotation} annotation
+ * @prop {Group} group - Group to which the annotation belongs
+ * @prop {boolean} isPrivate - Annotation is private?
```

Comment:
```suggestion
 * @prop {boolean} isPrivate - Is annotation private?
```

Although TBH the comment doesn't add value over the prop name, so you could also just remove it.